docs/essentials/overriding.md

```diff
@@ -0,0 +1,151 @@
+    <section class="bg-slate-200">
+      <HeaderLinks />
+    </section>
+    <HeaderMain />
+    <HeaderMenu />
+  </header>
+</template>
+
+<script setup lang="ts">
+import HeaderLinks from '#ioc/molecules/HeaderLinks'
+import HeaderMain from '#ioc/molecules/HeaderMain'
+import HeaderMenu from '#ioc/molecules/HeaderMenu'
+</script>
+```
+
+Finally, we have to import `module-b` after `module-a` in the `storefront-x.config.js` file.
+
+```js
+// storefront-x.*.config.js
+
+export default {
+  modules: [
+    // some modules...
+    'module-a',
+    // some modules...
+    'module-b',
+    // some modules...
+  ],
+}
+```
+
+The component from `module-b` will be used instead of the component from `module-a`.
+
+Vice versa, if we import `module-b` before `module-a`, the component from `module-a` will be used. And if we import only one module (`module-a` or `module-b`), the component from that module will be used. **The modules are independent of each other.**
+
+### Nested files
+
+The example above uses direct child of `organisms` concept. That is not a limitation, we can also override nested files. We just have to preserve the same path to the file in a new module. Let's look at the example, in which we want to change theme color to the one used by our company.
+
+::: code-group
+
+```html [module-a/base/templates/index.html]
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=2.0, minimum-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="apple-touch-icon" href="/apple-touch-icon.png" sizes="180x180" />
+    <meta name="theme-color" content="#FB923C" />
+  </head>
+  <body>
+    <div id="app"></div>
+    <script type="module" src="/entry.client.ts"></script>
+  </body>
+</html>
+```
+
+```html [module-b/base/templates/index.html]
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=2.0, minimum-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="apple-touch-icon" href="/apple-touch-icon.png" sizes="180x180" />
+    <meta name="theme-color" content="#86B049" />
+  </head>
+  <body>
+    <div id="app"></div>
+    <script type="module" src="/entry.client.ts"></script>
+  </body>
+</html>
+```
+
+:::
+
+`index.html` is located in the `base` concept and `templates` directory. So, we have to create the same path in the `module-b` and put the new file there.
+
+Then, if we use `module-b` after `module-a` in the `storefront-x.config.js` file, the `index.html` from `module-b` will be used instead of the `index.html` from `module-a`.
+
+You can find another practical example of overriding in the Cookbook section: [Override component](/cookbook/override-component).
+
+## Which concepts support overriding?
```

Comment:
I would add small descriptions in parentheses to explain what these concepts do

eg: components (contains .vue components)